components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -348,10 +350,24 @@ func (r *Reconciler) syncOauth2ClientIDAndSecret(ctx context.Context, backendSta
 		r.credentials.clientID = credentials.clientID
 		r.credentials.clientSecret = credentials.clientSecret
 		r.credentials.tokenURL = credentials.tokenURL
+		certsURL := computeCertsURL(string(credentials.tokenURL))
+		r.credentials.certsURL = []byte(certsURL)
 	}
 	return nil
 }
 
+// computeCertsURL computes the certs url from the given token url.
```

Comment:
Is it not good idea to add certs URL to the `eventing-webhook-auth` secret first? Then our code didn't this addition logic and no additional tests. It leads in general less work. Hence, I'd suggest to pause this.

Reply:
Changed to read the certs url directly from the secret.